Classes/Domain/Repository/PeriodRepository.php

```diff
@@ -64,8 +67,9 @@ public function findByDateAndBegin(\DateTime $dateTime, int $facilityUid): Query
     {
         $date = clone $dateTime;
         $date->setTime(0, 0);
-        $begin = clone $dateTime;
-        $begin->setDate(1970, 1, 1);
+        // $begin must be UTC because TCA saves that timestamp in UTC but others in configured timezone
+        $begin = new \DateTime('@0');
```

Comment:
@[timestamp] should always be the last possible option. Please have a look here: https://github.com/jweiland-net/events2/blob/master/Classes/Domain/Model/Event.php#L246-L254 Maybe that will match your needs.

Reply:
I am using this because TYPO3 saves `time` timestamps in UTC ([DataHandler](https://github.com/TYPO3/TYPO3.CMS/blob/master/typo3/sysext/core/Classes/DataHandling/DataHandler.php#L2526-L2531)). I could use something like `new \DateTime('1970/01/01', new \DateTimeZone('UTC'))` instead but it must be UTC

Reply:
I have nothing against UTC, but why are you working with 01.01.1970 always? Don't get it...

Reply:
This is because of that field just saves the `time` like 14:00. The real date is inside the field `date` but without the time so like 2021/06/10 00:00.